tests/phpunit/api/v3/GeometryTest.php

```diff
@@ -492,4 +535,68 @@ public function testGetDistance() {
     $this->assertEquals('2197', (int) $result['values']);
   }
 
+  /**
+   * Test returning spatial properties for a geometry
+   */
```

Comment:
I'd like to try to avoid as much as possible tests which may change from time to time.  I don't know how often NSW branch boundaries change, but it certainly happens in other states.  Ideally we'd construct a fake geometry ourselves with known coordinates for the purpose of testing, but if we're going to take a real-life boundary then can we make sure the one we choose almost never change?  Like a state boundary for example.

Reply:
@andrew-cormick-dockery well we effectively are doing that we have the geometries that are used in theses tests in the load subfolder so in effect they are fake so to speak so these tests will live beyond any redistribution